src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -413,6 +440,16 @@ static void _repeat(LottieGroup* parent, unique_ptr<Shape> path, RenderContext*
         mathTranslateR(&m, -repeater->anchor.x, -repeater->anchor.y);
 
         auto pm = PP(shape)->transform();
+
+        if (repeater->transform && !mathIdentity(static_cast<const Matrix*>(repeater->transform))) {
+            m = mathMultiply(repeater->transform, &m);
+            if (pm) {
+                Matrix inverse;
+                mathInverse(repeater->transform, &inverse);
+                *pm = mathMultiply(&inverse, pm);
```

Comment:
I know it's a bit expensive approach, but I don;t expect this scenario to happen very often